entries/n/n26.com.json

```diff
@@ -5,12 +5,22 @@
       "sms",
       "custom-software"
     ],
+    "custom-software": [
+      "N26 app"
+    ],
     "documentation": "https://support.n26.com/en-eu/security/passwords-and-codes/how-to-log-in-with-two-factor-authentication",
     "keywords": [
       "banking"
     ],
     "regions": [
-      "de"
+      "at",
+      "br",
+      "fr",
+      "de",
+      "it",
+      "es",
+      "ch",
+      "us"
```

Comment:
Given that one of the region choices on there is "Europe", this list is definitely not complete. Should we drop it or add the rest of the countries here?

Reply:
I've added region codes based on [this page](https://support.n26.com/en-eu/account-and-personal-details/opening-an-account/can-i-open-an-n26-account-in-my-country) in https://github.com/2factorauth/twofactorauth/pull/6296/commits/ff35dc57b4c372ac254bb805bd4bd207ac3e594f